src/v2/models/AppState.js

```diff
@@ -137,6 +123,18 @@ export default Model.extend({
     return reRender;
   },
 
+  showSignoutLinkInCurrentForm (hideSignOutLinkInMFA) {
```

Comment:
- maybe add comments to elaborate use cases, why we need complicated logic.?
- rename to some thing like `isXXX` or `shallXXX` (e.g. shallShowSignoutLink) ?